packages/applications-service-api/src/routes/documents.v3.js

```diff
@@ -7,6 +7,12 @@ const { asyncRoute } = require('@pins/common/src/utils/async-route');
 
 const router = express.Router();
 
+router.get(
+	'/:caseReference',
+	validateRequestWithOpenAPI,
```

Comment:
Suggestion / question for discussion - Is it worth doing custom validation on controller instead of the swagger validation for the `caseReference`? For example if we are sent `Rule 8 Letter` instead of `Rule 8 letter` it will return a 404 but we can easily capitalise the `caseReference` param and compare it a capitalised enum of "RULE 8 LETTER". This will ensure a small capitalisation issue isn't going to cause any issues. What do you guys think?